common/system/tasks/buzzer_heartbeat.h

```diff
@@ -1,13 +1,15 @@
-
 #ifndef COMMON_SYSTEM_TASKS_BUZZER_HEARTBEAT_H_
 #define COMMON_SYSTEM_TASKS_BUZZER_HEARTBEAT_H_
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#include "hardware_manager.h"
```

Comment:
This include can be moved to the cpp file. 

Generally, it's best practice to include only things that are necessary in the header file so that consumers of the file don't have to transitively include things they don't need.